cmake/Install.cmake

```diff
@@ -140,12 +140,100 @@ set(CPACK_IFW_PRODUCT_URL "https://www.energyplus.net")
 include(cmake/TargetArch.cmake)
 target_architecture(TARGET_ARCH)
 
+# Debug
+
+#cmake_host_system_information(RESULT _OS_NAME QUERY OS_NAME)
+#message("-- OS_NAME variable is set to: " ${_OS_NAME})
+
+#cmake_host_system_information(RESULT _OS_RELEASE QUERY OS_RELEASE)
+#message("-- OS_RELEASE variable is set to: " ${_OS_RELEASE})
+
+#cmake_host_system_information(RESULT _OS_VERSION QUERY OS_VERSION)
+#message("-- OS_VERSION variable is set to: " ${_OS_VERSION})
+
+#cmake_host_system_information(RESULT _OS_PLATFORM QUERY OS_PLATFORM)
+#message("-- OS_PLATFORM variable is set to: " ${_OS_PLATFORM})
+
+#message("CMAKE_SYSTEM_VERSION=${CMAKE_SYSTEM_VERSION}")
+#message("CMAKE_SYSTEM_NAME =${CMAKE_SYSTEM_NAME}")
+#if(MSVC)
+  #message("CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION=${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION}")
+  #message("MSVC_VERSION =${MSVC_VERSION}")
+#endif()
+
+# Mac
+#-- OS_NAME variable is set to: Mac OS X
+#-- OS_RELEASE variable is set to: 10.14.6
+#-- OS_VERSION variable is set to: 18G2022
+#-- OS_PLATFORM variable is set to: x86_64
+# CMAKE_SYSTEM_VERSION=18.7.0
+# CMAKE_SYSTEM_NAME =Darwin
+
+# Ubuntu
+#-- OS_NAME variable is set to: Linux
+#-- OS_RELEASE variable is set to: 5.4.0-42-generic
+#-- OS_VERSION variable is set to: #46~18.04.1-Ubuntu SMP Fri Jul 10 07:21:24 UTC 2020
+#-- OS_PLATFORM variable is set to: x86_64
+# CMAKE_SYSTEM_VERSION=5.3.0-51-generic
+# CMAKE_SYSTEM_NAME =Linux
+
+# Windows
+#-- OS_NAME variable is set to: Windows
+#-- OS_RELEASE variable is set to:  Professional
+#-- OS_VERSION variable is set to:  (Build 19041)
+#-- OS_PLATFORM variable is set to: AMD64
+# CMAKE_SYSTEM_VERSION=10.0.19041
+# CMAKE_SYSTEM_NAME=Windows
+# CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION=10.0.17763.0
+# MSVC_VERSION=1926
+
+#
+# End debug
+
+if(APPLE)
+  # Looking at cmake source code OS_RELEASE is already set to the output of `sw_vers -productVersion` which is what we want
+  cmake_host_system_information(RESULT OSX_VERSION QUERY OS_RELEASE)
+  message("-- OS_RELEASE variable is set to: " ${OSX_VERSION})
+  if(NOT CMAKE_OSX_DEPLOYMENT_TARGET STREQUAL "")
+    message("Using CMAKE_OSX_DEPLOYMENT_TARGET=${CMAKE_OSX_DEPLOYMENT_TARGET}")
+    set(OSX_VERSION "${CMAKE_OSX_DEPLOYMENT_TARGET}")
+  endif()
```

Comment:
Using CMAKE_OSX_DEPLOYMENT_TARGET if set, fall back to the system version. How's that @Myoldmopar ?

Reply:
:+1: